mmeds/documents.py

```diff
@@ -125,21 +127,35 @@ def generate_MMEDSDoc(self, name, doc_type, config, access_code):
                 data_type = self.reads_type + '_demuxed'
             # Handle all sequences in one file
             else:
-                # Create links to the files
-                (new_dir / 'barcodes.fastq.gz').symlink_to(self.files['barcodes'])
-                (new_dir / 'for_reads.fastq.gz').symlink_to(self.files['for_reads'])
-
-                # Add the links to the files dict for this analysis
-                files['barcodes'] = new_dir / 'barcodes.fastq.gz'
-                files['for_reads'] = new_dir / 'for_reads.fastq.gz'
-
-                # Handle paired end sequences
-                if self.reads_type == 'paired_end':
+                if 'solo' in self.barcodes_type:
```

Comment:
Is there a spot in the code where this type of conditional will cause an error if it's 'single' rather than 'solo'? I think it is better to use 'single' as that's the terminology used elsewhere for these types of situations. I don't think I use 'solo' anywhere else so just doing a search and replace in each file should be safe